### ISSUE TITLE
Unable to build on Windows

### ISSUE THREAD

**Issue by n00kii** [ParadoxSpiral/mpv-rs#26]
`README` says
> For windows targets this is expected to be already built, with a directory named MPV_SOURCE/64 or /32 containing [build artifacts](https://mpv.srsfckn.biz/) for 64-bit and 32-bit targets respectively.

but the link provided is dead. Unsure exactly what "build artifacts" is supposed to be, do I need to build mpv myself from source? (Kinda sucks since the process is a little convoluted on Windows)

There are multiple downloads of different forms of mpv available, can I use one of those somehow? For example, there's

- the mpv dev lib
![image](https://user-images.githubusercontent.com/57325298/184989050-9c80864b-7136-4648-8742-8f6696f78262.png)
- the raw mpv source 
![image](https://user-images.githubusercontent.com/57325298/184989303-552b4a97-666c-4e0a-8c94-7d6b07765e17.png)
- mpv-build
![image](https://user-images.githubusercontent.com/57325298/184989516-4dd0ca2a-988a-48dd-9561-9b894f7a3692.png)

For reference, I've already tried pointing the environment variable to each of these directories, nested inside a `64` folder as per the `README` but `cargo` complains that 
```shell
= note: LINK : fatal error LNK1181: cannot open input file 'mpv.lib'
```

**Comment by n00kii** [comment]
Managed to build `mpv`, but there is still no "`mpv.lib`" file anywhere. Am I missing something?

![image](https://user-images.githubusercontent.com/57325298/184995632-309915a4-fc17-4871-a3a0-3962d7bbb27b.png)

**Comment by moxun33** [comment]
same error.

**Comment by olehbozhok** [comment]
I build mpv.lib using https://github.com/Ivshti/libmpv
but have error 
(exit code: 0xc0000135, STATUS_DLL_NOT_FOUND)

**Comment by ParadoxSpiral** [comment]
Yeah the situation seems to have changed a bit. I will try getting the build to work on Windows, only got access to a Windows machine recently.

**Comment by prayag17** [comment]
Any updates on this?

**Comment by maulik9898** [comment]
Also facing same issue `error: process didn't exit successfully: target\debug\mpv-rs.exe (exit code: 0xc0000135, STATUS_DLL_NOT_FOUND)`

**Comment by voidentente** [comment]
- Make sure you have the most recent Visual Studio installed (see https://github.com/mpv-player/mpv/blob/master/DOCS/compile-windows.md#linking-libmpv-with-msvc-programs saying that it must support `stdint.h`
- You can download the libmpv stuff from https://sourceforge.net/projects/mpv-player-windows/files/libmpv/, or build it yourself
- Latest libmpv-rs expects libmpv API version 1.108, but the latest files are of a different major version, so use mpv-dev-x86_64-20201220-git-dde0189.7z or any other old archive that still uses major version 1 (see https://github.com/mpv-player/mpv/blob/master/DOCS/client-api-changes.rst for the API versions)
- Build the .lib file by running
```
CALL "C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Auxiliary\Build\vcvarsall.bat" x64
lib /def:mpv.def /name:mpv-1.dll /out:mpv.lib /MACHINE:X64
```
or similar. If warnings pop up like "...not supported on this platform...", it's because mpv.def is missing the EXPORTS statement, which you can just add as the first line (see https://github.com/mpv-player/mpv/issues/5007)
- Put the .lib file in the project root or anywhere where the linker can find it
- Put the .dll file in the directory of the executable or in any directory in PATH

**Comment by nini22P** [comment]
Rename `libmpv.dll.a` to `mpv.lib`
